tests/vmi_networking_test.go

```diff
@@ -754,7 +754,7 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				}
 			})
 
-			It("preserves connectivity", func() {
+			It("[Conformance] preserves connectivity", func() {
```

Comment:
On this one I have a question, since it relies on a feature gated feature (migrations). Will we enable migrations in this case, or will we just skip the test?

Reply:
Are you asking what would happen on a cluster with disabled migrations when somebody runs the conformance suite or asking what should be our behaviour in that case?

I think it is ok to let all features that are disabled be skipped, i.e. let user decide what they want to enable and then just make sure that everything enabled works as expected with the suite. We can't require it to be enabled (if we would, we can remove the feature gate, right?).

Reply:
If it just gets skipped, all is good from my side.

Reply:
Let me confirm that.

Reply:
Well, it fails:

```
Plugin: kubevirt-conformance
Status: failed
Total: 622
Passed: 8
Failed: 1
Skipped: 613

Failed tests:
[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]Networking VirtualMachineInstance with masquerade binding mechanism when performing migration [Conformance] preserves connectivity
```

I will add a skip to the test.

Reply:
I added the skip:

```
Plugin: kubevirt-conformance
Status: passed
Total: 622
Passed: 8
Failed: 0
Skipped: 614
```